src/Compiler/Driver/ParseAndCheckInputs.fs

```diff
@@ -1197,6 +1198,9 @@ let CheckOneInputAux
 
     cancellable {
         try
+            use _ =
+                Activity.Start "ParseAndCheckInputs.CheckOneInput" [| "inputName", inp.FileName |]
```

Comment:
Should `inputName` and `fileName` be actually the same field names? The only reason they are different I think is because I looked at the method name and chose `input` here rather than `file`.

Reply:
Seeing it again, I think it is better (from the perspective of looking at provided traces) to use same names across callsites, even if the arguments may vary slightly.
So well known names like:

fileName
project

Should be always the same.